src/client/coupons.rs

```diff
@@ -16,9 +16,14 @@ pub struct Coupon {
     pub id: String,
     /// User-facing coupon code
     pub redemption_code: String,
+    // The number of times this coupon has been redeemed.
```

Comment:
nit: can you make the comments on the field use a triple slash? I'm not sure why it's not showing up for you but on my side when it's not I get a warning like `missing documentation for a struct field`